packages/api/src/compile-schemas.js

```diff
@@ -1,6 +1,6 @@
 import Ajv from "ajv";
 import pack from "ajv-pack";
-import { safeLoad as parseYaml } from "js-yaml";
+import { safeLoad as parseYaml, safeDump as serializeYaml } from "js-yaml";
```

Comment:
Is `safeDump` supposed to be imported?

Reply:
Yeah it's used to write the merged yaml now!